src/main/webapp/scripts/map.js

```diff
@@ -49,31 +69,92 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
-  /*Add Hard-Coded Markers */
+  
+  /* Add Hard-Coded Markers */
   addMarker(map);
 }
 
-/* Temporary Function to Hold Marker for Sky Garden */
 function addMarker(map) {
-  const skyGarden = {lat: 51.510881, lng: -0.083751}
-  const marker = new google.maps.Marker({
-    position: skyGarden,
-    map: map,
-    title: "Sky Garden"
-  })
-  marker.addListener("click", () => {
+  /* Hard-Coded Marker Data */
+  var placeName;
+  var placeLatLng;
+  var placeCategory;
+  var placeRecommendation;
+
+  /* Iterate through stored recommendations to get details */
+  for (i = 0; i < recommendations.length; i++) {
+    var rec = recommendations[i]
+    placeCategory = rec[0];
+    placeName = rec[1];
+    placeLatLng = new google.maps.LatLng(rec[2], rec[3]);
+    placeRecommendation = rec[4];
+    const marker = new google.maps.Marker({
+      position: placeLatLng,
+      map: map,
+      title: placeName,
+      icon: getColourMarker(placeCategory)
+    })
+    /* Add Listener for Click on Marker */
+    google.maps.event.addListener(marker, "click", () => {
+    /* Update the HTML */
+    var markerRec = getRecommendationDetails(marker.title);
+    document.getElementById("category-header").innerHTML = markerRec[0];
```

Comment:
isnt markerRec[0] the same as recommendation[0] ?
and similarly with other markerRec[i] with recommendation[i]